waku/v2/protocol/waku_swap/waku_swap.nim

```diff
@@ -241,6 +241,11 @@ proc init*(wakuSwap: WakuSwap) =
     else:
       info "Payment threshhold not hit"
 
+
```

Comment:
This is now just whitespace so shouldn't be part of the PR. Before you commit, go through the git diff to ensure that you're only committing _code_ changes. One or two formatting improvements are OK (like adding whitespace) if well motivated.